tools/clang-format.sh

```diff
@@ -46,13 +46,13 @@ function main() {
   CURL=${CURL:-curl}
 
   # Default to sha256sum, but honor the env variable just in case
-  if [ $(which sha256sum) ] ; then
+  if [ -n "$(which sha256sum)" ] ; then
     SHASUM=${SHASUM:-sha256sum}
   else
     SHASUM=${SHASUM:-shasum -a 256}
   fi
 
-  ARCHIVE=$ROOT/$(basename ${URL})
+  ARCHIVE="$ROOT/$(basename "${URL}")"
```

Comment:
Do these two nested quotes work?

Reply:
Yep, because the inner quotes are in a subshell.